server/test/views/components/TextFormatterTest.java

```diff
@@ -82,21 +82,67 @@ public void rendersRequiredIndicator() {
   }
 
   @Test
-  public void correctlyInsertsRequiredIndicatorBeforeList() {
-    ImmutableList<DomContent> content =
+  public void insertsRequiredIndicatorBeforeLists() {
+    ImmutableList<DomContent> contentWithUnorderedList =
         TextFormatter.formatText(
             "Here is some text.\n" + "* list item one\n" + "* list item two",
             /* preserveEmptyLines= */ false,
             /* addRequiredIndicator= */ true);
-    String htmlContent = content.get(0).render();
+    String htmlContentWithUnorderedList = contentWithUnorderedList.get(0).render();
+
+    assertThat(htmlContentWithUnorderedList)
```

Comment:
question (non-blocking):  Why makes separate assertions rather than a single `equals` assertion with the full html content of the static question?  Seems like that would have the added benefit of verifying that everything is in the right place.

Reply:
That's a great question. I remember there was something tricky about getting the syntax right for checking it all at once, but I'll try again. If it doesn't work, I'll add a comment explaining why it's tested like this.

Reply:
Welp, I'm not sure what I was doing before but changing these to `isEqualTo` was super easy! Updated.